### PR TITLE
feat: create static selection component

### PR DESCRIPTION
**Pull request by lwhiteley**
- [x] fix is selected state

### CODE REVIEW

**Comment by lwhiteley** [comment]
@iambumblehead  also remember to enable auto deleting of branches after merge

**Comment by iambumblehead** [comment]
The auto-delete setting is enabled now

File: README.md [hunk 1]
```diff
@@ -6,17 +6,30 @@
...
   value="one"
-  onChange={() => console.log('change!')}
-  onClose={() => console.log('close!')}
-  onOpen={() => console.log('open!')} />;
+  onChange={(value) => console.log('change!', value)}
+  onClose={(closedBySelection) => console.log('closedBySelection?:', closedBySelection)}
+  onOpen={() => console.log('open!')}
+/>;
+
+// static selection
```
[the remark this answers is not in the document]
Comment (iambumblehead) [reply README.md]: maybe a description like "static selection, dropdown selection is always open and does not close"

[the remark this answers is not in the document]
Comment (lwhiteley) [reply README.md]: technically its not always open, it is a rendered selection component.
can be used within other components like popovers etc.

for. eg https://codesandbox.io/s/thexv?module=/example.js

[the remark this answers is not in the document]
Comment (iambumblehead) [reply README.md]: Maybe a better description would be "use the Selection component with other components like popovers etc."

[the remark this answers is not in the document]
Comment (lwhiteley) [reply README.md]: updated

[the remark this answers is not in the document]
Comment (iambumblehead) [reply README.md]: :ship: :+1:

**Comment by iambumblehead** [comment]
approved :)

**Pull request merged**